src/lib/headers/CustomHeader.js

```diff
@@ -64,7 +65,7 @@ export class CustomHeader extends React.Component {
     return false
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
I've just added this flag to keep this PR atomic. Happy to fix this in an subsequent PR!